<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1155</width>
    <height>735</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Face Features Based Cursor Controller</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QGroupBox" name="controlPanelGroupBox">
      <property name="title">
       <string>Panel de control</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="2">
        <widget class="QCheckBox" name="controlMouseCheckbox">
         <property name="text">
          <string>Controlar el mouse</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="calibrateButton">
         <property name="text">
          <string>Reestablecer puntos</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QCheckBox" name="showProcessingCheckbox">
         <property name="text">
          <string>Mostrar procesamiento</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="cameraGroupBox">
      <property name="title">
       <string>Cámara</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="CursorControllerWidget" name="cursorControllerWidget" native="true"/>
       </item>
       <item row="0" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QGroupBox" name="graphicsGroupBox">
      <property name="title">
       <string>Gráficos</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="3" column="1">
        <widget class="Grapher" name="velocityGrapher" native="true"/>
       </item>
       <item row="1" column="1">
        <widget class="Grapher" name="positionGrapher" native="true"/>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="positionLabel">
         <property name="text">
          <string>Posición de target</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="velocityLabel">
         <property name="text">
          <string>Velocidad de target</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="0">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1155</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="calibrateAction">
   <property name="text">
    <string>Calibrar</string>
   </property>
  </action>
  <action name="showProcessingAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Mostrar procesamiento</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CursorControllerWidget</class>
   <extends>QWidget</extends>
   <header>cursorcontrollerwidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Grapher</class>
   <extends>QWidget</extends>
   <header>grapher.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
